src/test/java/me/karakelley/tictactoe/EasyBehaviorTest.java

```diff
@@ -0,0 +1,49 @@
+package me.karakelley.tictactoe;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.spy;
+
+class EasyBehaviorTest {
+
+  BoardState boardState = spy(new BoardState());
+  Player player;
+
+  @BeforeEach
+  public void setUp() {
+    player = new ComputerPlayer("X", new EasyBehavior());
+  }
+
+  @Test
+  public void testMove() {
+   int cell = Integer.parseInt(player.pickMove(boardState));
+   assertTrue(cell < 9 );
+  }
+
+  @Test
+  public void testChoseCellNotAlreadyTaken() {
```

Comment:
Typically for class like this which have some level on non-determinism, I'll take one of two approaches:

1. Inject the non-deterministic bit (or something like makes it deterministic such as the [seed](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#Random-long-))

2. Do what you did here, except run it a handful of times to be super sure...

```
for (int i = 0; i < 100; i++) {
  // test body here...
}
```